config/environments/test.rb

```diff
@@ -60,6 +60,7 @@
   config.action_mailer.default_url_options = { host: 'localhost', port: 3000 }
 
   config.api_url = "http://www.example.com"
+  config.tab_user_url = "http://www.example.com"
```

Comment:
Should the `api_url` include `/api/v1/` here?

Reply:
It doesn't matter that much, this is not actually fetched anywhere.